pandas/io/formats/html.py

```diff
@@ -77,9 +79,20 @@ def _write_cell(self, s, kind='td', indent=0, tags=None):
                                ('>', r'&gt;')])
         else:
             esc = {}
+
         rs = pprint_thing(s, escape_chars=esc).strip()
-        self.write(u'{start}{rs}</{kind}>'
-                   .format(start=start_tag, rs=rs, kind=kind), indent)
+
+        if self.render_links and _is_url(rs):
+            rs_unescaped = pprint_thing(s, escape_chars={}).strip()
+            start_tag += '<a href="{url}" target="_blank">'\
```

Comment:
Instead of the explicit line continuation can you just move `.format(` to the line above and use the implicit continuation on next line? Typically how we handle in code base